lib/routes/futunn/highlights.js

```diff
@@ -32,25 +40,24 @@ module.exports = async (ctx) => {
 
             const $ = cheerio.load(response.data);
 
-            const description = $('div.newsDetailBox div.main div.inner').html() || '原文已被删除';
+            const description = $('div.newsDetailBox div.main div.inner').html();
 
-            const date = $('div.timeBar').text();
+            const date = $('div.timeBar').children().remove().end().text().trim();
 
             const single = {
                 title: title,
                 link: itemUrl,
                 description: description,
-                pubDate: date_util(date, 8),
+                pubDate: date_parser(date, 'YYYY/MM/DD HH:mm')
```

Comment:
I would suggest something like:

```js
const timezone = require('@/utils/timezone');
const { parseDate } = require('@/utils/parse-date');

// ...

const single = {
    // ...
    pubDate: timezone(parseDate(date, 'YYYY/MM/DD HH:mm'), +8)
}
```

Reply:
I manually check that the timezone shows is correct. The website is in traditional Chinese which returns UTC+8 as default. Is it necessary to be guarded with timezone util?

Reply:
You probably misunderstood the usage of `timezone`. I suppose you are using a PC for development, with UTC+8 as the default timezone. If some other user deploys RSSHub on a server with UTC+6 as the default timezone, the date string will be wrongly interpreted. And this is why we introduce the `timezone` utility.

Reply:
although not quite sure what is the difference. I revised again. Please check. thanks~~~